pug-vdom.js

```diff
@@ -154,7 +154,7 @@ Compiler.prototype.visitEach = function (node, parent) {
   this.addI(`var ${node.val} = ${node.obj}[${key}]\r\n`)
   this.visitBlock(node.block)
   this.indent--
-  this.addI(`})\r\n`)
+  this.addI(`}.bind(this))\r\n`)
```

Comment:
You could consider a fat arrow function here instead? Do we allow that in our code?

Reply:
Well at present the code maintains compatibility with < es6 browsers, but those are becoming increasingly irrelevant. I would say if we make the jump into ES+ features, we would need to at least bump the major version. 

I think for this PR bind gets the job done, but maybe we can evaluate a more sweeping ES upgrade in a new major version. I'd be happy to look into that when I find a moment

Reply:
So... no need to change here?

Reply:
Yes, let's keep the bind. No reasons to break the compatibility

Reply:
@albb0920 could you resolve the conflicts with master?